Jenkinsfile

```diff
@@ -86,7 +86,7 @@ def releaseBuilder(){
     """
   }
   stage('Release Docker Image'){
-    def dockerVersion = "csd-2.3.1-${spark_version.split('-')[-1]}"
+    def dockerVersion = "2.3.1-csd-${spark_version.split('-')[-1]}"
```

Comment:
Ummm... you're reconstructing `spark_version` -- just use it directly. For release builds, the `SNAPSHOT` will never be part of `pom.version`, so you could even use that instead of making `spark_version`. The only question with that is whether we'll want `SNAPSHOT` in the docker tag for PR builds (which don't use the docker image now, but may at some point.)

Reply:
Ah great point. I would like it to be labeled as -SNAPSHOT for PR builds